### DISCUSSION TITLE
Administrator permission in roles & Ability to drag roles

### DISCUSSION THREAD

**Discussion by I2rys** [stoatchat/discussions#543] [Feature Suggestions]
It's a big pain enabling all the permission in "Administrator" or "Admin" roles so why not add a "Administrator" permission like Discord? Also it would be much easier to organize the roles If we can just drag them. :)

**Comment by nulldg** [comment]
two suggestions should be split into two threads.

**Comment by nulldg** [comment]
as for "administrator", i would simply add a button to set every permission and a button to reset them. this is functionally equivalent, and probably the reason there is no administrator permission.

**Reply by I2rys** [reply]
I guess that sounds differently from Discord and that works too.
